src/main/java/gov/cabinetofice/gapuserservice/dto/UserSubsRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package gov.cabinetofice.gapuserservice.dto;
+
+import lombok.Builder;
+
+import java.util.List;
+
+@Builder
+public record UserSubsRequestDto (List<String> userSubs) {
```

Comment:
Might be useful to return both subs & emails

Reply:
This is the request dto. It doesn't contain any emails?